server/config/index.js

```diff
@@ -6,11 +6,12 @@ module.exports = {
       debug: false,
       debugIORedis: false,
     },
-    connections: {
+    connections: {   
       default: {
         connection: {
-          host: '127.0.0.1',
-          port: 6379,
+          socket:{
```

Comment:
Socket connection types shouldn't be the default, optional but not default.

Reply:
Updated from default to optional